docs/_writing-tests/lint-tool.md

```diff
@@ -58,6 +58,9 @@ below to fix all errors reported.
   element whose `content` attribute has a malformed value; **fix**: ensure
   the value of the `content` attribute starts with `?` or `#` or is empty.
 
+* **MISSING-LINK**: CSS test file is missing a link to a spec. **fix**: Ensure that there is a `<link rel="help" src="[url]">` for the spec.
+  * Note: `MISSING-LINK` is designed to ensure that the CSS build tool can find the tests. Note that the CSS build system is primarily used by [test.csswg.org/](http://test.csswg.org/), which doesn't use `wptserve`, so `*.any.js` and similar tests won't work there; stick with the `.html` equivalent.
```

Comment:
I don't think the comment about wptserve is terribly useful; we don't lint for other things that may break when you're not using it.

Reply:
Well, actually, what lead me to be frustrated by these missing docs:
- The only way to resolve the lint is with a `<link` tag; (see #5770)
-`.any.js` uses `// META: script`, but doesn't have a `// META: link` or `// META: spec`
- Even _if_ we [added](https://github.com/w3c/web-platform-tests/pull/10289) `// META: spec` for `any.js`, it doesn't work outside `wptserve`, so we'd have to change the build rule
- The lint is for the build rule in the first place

So, without that comment, you dive down a rabbit hole of "how do I fix this lint in my `.any.js` test?" instead of "dumb, but I'll just use the `.html` equivalent test."